content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java

```diff
@@ -2038,6 +2056,7 @@ private News addNewArticleVersionWithLang(News news, Identity versionCreator) th
       if (draftPage != null) {
         deleteDraftArticle(draftPage.getId(), draftPage.getAuthor());
       }
+      NewsUtils.broadcastEvent(NewsUtils.ADD_ARTICLE_TRANSLATION, versionCreator, news);
```

Comment:
to be discussed

Reply:
This is to refresh the attached activity cache after adding or deleting an article translation!